packages/core/integration-tests/test/css.js

```diff
@@ -33,6 +33,20 @@ describe('css', function() {
     assert.equal(output(), 3);
   });
 
+  it('should include a banner when supplied', async function() {
+    const bannerString = 'Welcome to my bundle, v0';
+    let b = await bundle(path.join(__dirname, '/integration/css/index.js'), {
+      banner: bannerString
+    });
+
+    await run(b);
+    let file = await fs.readFile(
+      path.join(__dirname, '/dist/index.css'),
+      'utf8'
+    );
+    assert(file.startsWith(`// ${bannerString} \n`));
```

Comment:
css should probably use: 
```css
/* comment */
```

Reply:
thanks, that's a good point! actually now that i'm thinking about it some more, i think all the packagers should use the multiline comment markup. If the developer passes a multiline banner string, it would break the current implementation for JS and CSS.

Reply:
fixed up in c3c9bf4. @DeMoorJasper let met know of any more feedback/next steps!